### PR TITLE
use central package management

### PR DESCRIPTION
**Pull request by Bertk**
- use central package management

### CODE REVIEW

File: .gitpod.yml [hunk 1]
```diff
@@ -11,9 +11,10 @@ tasks:
       "$DOTNET_ROOT/dotnet-install.sh" --channel 3.1 --install-dir "$DOTNET_ROOT"
       "$DOTNET_ROOT/dotnet-install.sh" --channel 5.0 --install-dir "$DOTNET_ROOT"
```
Comment (coderpatros) [inline .gitpod.yml:12]: You can drop these versions of the SDK from the gitpod config too

File: semver.txt [hunk 2]
```diff
@@ -1 +1 @@
-2.7.0
+3.0.0
```
Comment (coderpatros) [inline semver.txt:1]: Don't update this yet. I'll bump the version after merge and before release.

**Review by coderpatros — CHANGES_REQUESTED** [review]

**Comment by Bertk** [comment]
@coderpatros I resolved all the comments. The tests for my local build are green ;-)
image

**Comment by Furbyv** [comment]
Any updates on this?

**Comment by Bertk** [comment]
> Thanks for the PR @Bertk, can you do a find and replace on all the `FileInfo.FromFilename` calls to address the build warnings too?

Please review again. Done.

**Comment by tomkuipers** [comment]
Any updates on this?

**Pull request closed without merging**